main.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_ses_domain_identity" "domain" {
+  domain = "${var.domain}"
+}
+
+resource "aws_route53_record" "domain_amazonses_verification_record" {
```

Comment:
I would leave setting the verification record out of this module. Not all customers will have their DNS hosting on R53.

Reply:
Good point, but then the module is even more "useless". I'll add a flag maybe?

Reply:
I'd set it only if we pass the `zone_id`

Reply:
Done.

Reply:
@duboisph I approved, but can you add to the docs that the verification record is added when you pass the zone id?

Reply:
It is...

Reply:
> * \[`zone_id`\]: String(optional): Route 53 zone ID for the SES domain verification. If the `zone_id` is not set, we do not create verification record in R53.

;)